DevKit/DevKit/Classes/Data Structures/Linked List/LinkedList.swift

```diff
@@ -0,0 +1,175 @@
+//
+//  LinkedList.swift
```

Comment:
Can you also add documentation to the read me explaining, some of the public functions and how you would use something like this? Its kinds self explanatory but I think it would be good to have documentation around everything we add to this repo.

Reply:
I agree, added a more in-depth documentation, please let me know what you think.